commands_common/flags.go

```diff
@@ -4,10 +4,10 @@ import "github.com/urfave/cli/v2"
 
 func PromptFlag() cli.Flag {
```

Comment:
for some of the cases prompt is not optional, so most likely you will need to pass bool as input here and than either make it required or not